lib/doorkeeper/openid_connect/request_extension.rb

```diff
@@ -0,0 +1,16 @@
+module Doorkeeper
+  module Request
+    class << self
+      def get_strategy_with_multitype(grant_or_request_type, available)
+        grant_array = grant_or_request_type.split(" ").sort
+        if ["id_token", "token"] == grant_array
+           Doorkeeper::Request::IdTokenToken
+        else
+          get_strategy_without_multitype(grant_or_request_type, available)
+        end
+      end
+      alias_method :get_strategy_without_multitype, :get_strategy
+      alias_method :get_strategy, :get_strategy_with_multitype
+    end
+  end
+end
```

Comment:
I think this can be removed now with https://github.com/doorkeeper-gem/doorkeeper/pull/1028 merged?

Reply:
if we remove it, this gem will only be compatible with Doorkeeper 4.3.0 (latest version). If that's ok I can remove it.

Reply:
Actually I decided that it makes sense to drop this code and support only new version of Doorkeeper so we don't need to support legacy stuff.

Reply:
@nashby yeah I agree, thanks!